wcivf/apps/people/migrations/0028_person_pk_to_int.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-08-29 10:22
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def drop_varchar_pattern_ops_index(apps, schemaEditor):
+    models = [
+        apps.get_model("mentions", "mention_people"),
+        apps.get_model("people", "AssociatedCompany"),
+        apps.get_model("people", "Person"),
+        apps.get_model("peoplecvs", "CV"),
+        apps.get_model("pledges", "CandidatePledge"),
+        apps.get_model("leaflets", "Leaflet"),
+
+    ]
+    mp = apps.get_model("mentions", "mention_people")
+    index_names = [
+        x
+        for x in schemaEditor._constraint_names(mp)
+        if 'mentions_mention_peo_person_id_' in x
+    ]
+    for index_name in index_names:
+        schemaEditor.execute(
+            schemaEditor._delete_constraint_sql(
+                schemaEditor.sql_delete_unique,
+                mp,
+                index_name
+            )
+        )
+
+    for model in models:
+        index_names = schemaEditor._constraint_names(model, index=True)
+        for index_name in index_names:
```

Comment:
This took a while to understand because in this function we are re-using `index_names` and iterating `for index_name in index_names` twice for 2 different reasons. For clarity this could probably be 2 functions rather than one.